woocommerce/woocommerce
# Add Tiktok to OBW grow list

### All Submissions:






### Changes proposed in this Pull Request:

This PR needs to land after 14609-gh-Automattic/woocommerce.com

### How to test the changes in this Pull Request:

With a new site,

1. Disable "Display suggestions within WooCommerce " in Settings > Advanced > WooCommerce.com.
2. Go through the Onboarding Wizard, select 'United States' as your location
3. Navigate to the Free Features section in the Onboarding Wizard.
4. See that TikTok is listed.
5. Select TikTok and complete the Onboarding Wizard.
6. On the WordPress Plugins page, confirm that TikTok for WooCommerce plugin is installed.

### Other information:




### FOR PR REVIEWER ONLY:
